innovation_sweet_spots/pipeline/fetch_daps1_data/cb_utils.py

```diff
@@ -0,0 +1,64 @@
+import logging
```

Comment:
Kind of confused by these modules - does a lot of this (esp `fetch_daps_table`) not reproduce what is in `getters.inputs`? If the data is going to be filtered anyway, it would be better practice to do it once to avoid proliferating local data files